modules/server/nginx/searchbroker.conf.tmpl

```diff
@@ -4,10 +4,7 @@ server {
     include /etc/nginx/conf.d/acme-challenge.global;
 
     location / {
-        proxy_pass http://searchbroker-ui:8080/;
-        proxy_set_header X-Real-IP $remote_addr;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header Host $http_host;
+        rewrite ^ https://search.germanbiobanknode.de$request_uri? permanent;
```

Comment:
why not `https://$server_name$request_uri`

have you tested, that the access to .well-known still works with HTTP?

Reply:
Thanks, `https://$server_name$request_uri` is better, yes it's tested, see also http://search.germanbiobanknode.de/.well-known/acme-challenge/test.txt